test_scripts/Defects/5_1/980_SDL_responds_GENERIC_ERROR_instead_of_INVALID_DATA_when_soft_button_has_Type_is_Image_or_Both_and_Text_is_whitespace_or_\t_or_\n_or_empty.lua

```diff
@@ -0,0 +1,83 @@
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/980
+-- Description: [APIs] AlertManeuver: SDL responds GENERIC_ERROR instead of INVALID_DATA when soft button has Type is
+-- Image or Both and Text is whitespace or \t or \n or empty
+-- Precondition:
+-- 1) AlertManeuver is allowed by policy for "default" group.
+-- 2) SDL and HMI are started.
+-- 3) App is connected and at the FULL.
+-- In case:
+-- 1) Send AlertManeuver sure that soft button has Type is Image or Both and Text is whitespace or \t or \n or empty.
+-- Expected result:
+-- 1) SDL must respond with resultCode "INVALID_DATA" and success:"false".
+-- Actual result:
+-- 1) SDL responds with resultCode "GENERIC_ERROR" and success:"false".
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local utils = require('user_modules/utils')
+local json = require("modules/json")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+  type_BOTH = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "BOTH", text = "Close", softButtonID = 821, } } -- soft button has type is "BOTH"
+  },
+  type_IMAGE = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "IMAGE", text = "Close", softButtonID = 821, } } -- soft button has type is "Image"
+  },
+  text_HorizontalTab = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "Close\t", softButtonID = 821, } } -- soft button has Text is \t
+  },
+   text_Whitespace = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "        ", softButtonID = 821 } } -- soft button has Text is whitespace
+  },
+  text_empty = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "", softButtonID = 821 } } -- soft button has Text is empty
+  },
+  text_lineBreak = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "Close\n", softButtonID = 821 } } -- soft button has Text is \n
+  }
+}
+
+--[[ Local Functions ]]
+local function updatePreloadedPT(pGroups, pAppId)
+  local pt = common.sdl.getPreloadedPT()
+  pt.policy_table.functional_groupings["DataConsent-2"].rpcs = utils.json.null
+  pt.policy_table.app_policies[common.app.getParams(pAppId).fullAppID] = utils.cloneTable(pt.policy_table.app_policies.default)
+  pt.policy_table.app_policies[common.app.getParams(pAppId).fullAppID].groups = { "Base-4", "Navigation-1" }
+  pt.policy_table.functional_groupings["NewTestCaseGroup"] = pGroups
+  pt.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
```

Comment:
NewTestCaseGroup and pGroups are empty, so please remove redundant line

Reply:
@GetmanetsIrina You can find the updates in the commit dac0e07